spec/integration/node/support/keep-alive.js

```diff
@@ -0,0 +1,21 @@
+import createManifest from 'spec/integration/support/manifest'
+
+import forge, { configs } from 'src/index'
+
+export default function keepAlive(host, gateway) {
+  return {
+    verifySockets: (socketsReference) => {
+      const socketOrigins = Object.keys(socketsReference)
+      expect(socketOrigins.length).toBe(1)
+      const sockets = socketsReference[socketOrigins[0]]
+      expect(sockets.length).toBe(1)
+      const socket = sockets[0]
+      expect(socket.listenerCount('lookup')).toBe(1)
+      expect(socket.listenerCount('connect')).toBe(1)
+    },
```

Comment:
```suggestion
    verifySockets: (count, socketsReference) => {
      const socketOrigins = Object.keys(socketsReference)
      if (count === 0) {
        expect(socketOrigins.length).toBe(0)
        return
      }
      expect(socketOrigins.length).toBe(1)
      const sockets = socketsReference[socketOrigins[0]]
      expect(sockets.length).toBe(1)
      const socket = sockets[0]
      expect(socket.listenerCount('lookup')).toBe(count)
      expect(socket.listenerCount('connect')).toBe(count)
    },
```


Can we add an argument to the helper to state how many `listenerCount` we are expecting?

Then we can in both tests show that we expect 0 on the one and 1 on the other which makes it a little bit better and perhaps more helpful?

```ts
        keepAliveHelper.callApiTwice().then(() => {
          keepAliveHelper.verifySockets(1, httpAgent.freeSockets)
          keepAliveHelper.verifySockets(0, httpAgent.sockets)
          done()
```